src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -60,8 +60,8 @@ async function boot(userOptions?: Partial<CircuitStartOptions>): Promise<void> {
   const circuit = new CircuitDescriptor(components, appState || '');
 
   // Configure navigation via SignalR
-  Blazor._internal.navigationManager.listenForNavigationEvents((uri: string, intercepted: boolean): Promise<void> => {
-    return connection.send('OnLocationChanged', uri, intercepted);
+  Blazor._internal.navigationManager.listenForNavigationEvents((uri: string, state: string | undefined, intercepted: boolean): Promise<void> => {
+    return connection.send('OnLocationChanged', uri, history.state, intercepted);
```

Comment:
Why not using the `state` parameter here and in [WebViewIpcSender.ts](https://github.com/dotnet/aspnetcore/pull/42534/files#diff-f47c1c7ea89336f207a8d94e2bc6a854c8e9079a963015ccfa05b5ebb1f1c6f8) so that it is consistent with [Boot.WebAssembly.ts](https://github.com/dotnet/aspnetcore/pull/42534/files#diff-ec8d15c71f90fae0b6b64916781c3b2f0725b7d0c60fcaceee4172449ad60fa2)?
```suggestion
    return connection.send('OnLocationChanged', uri, state, intercepted);
```